JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -47,6 +48,23 @@ public async Task DispatchMessage(SQSMessage message)
             try
             {
                 typedMessage = _serialisationRegister.DeserializeMessage(message.Body);
+
+                typedMessage.MessageAttributes = message.MessageAttributes?.ToDictionary(
+                    source => source.Key,
+                    source =>
+                    {
+                        if (source.Value == null)
+                        {
+                            return null;
+                        }
+
+                        return new JustSaying.Models.MessageAttributeValue
+                        {
+                            StringValue = source.Value.StringValue,
+                            BinaryValue = source.Value.BinaryValue?.ToArray(),
```

Comment:
Correct that null goes to null, rather than null goes to empty array?

Reply:
yep, null will be propagated here